series/views.py

```diff
@@ -88,3 +88,14 @@ def delete(self, request, show_id: int):
         show = APIShow.objects.filter(id=show_id).first()
         show.followers.remove(request.user)
         return HttpResponse(200)
+
+class About(View):
```

Comment:
Consider using [TemplateView](https://docs.djangoproject.com/fr/2.0/ref/class-based-views/base/#django.views.generic.base.TemplateView)? It's a generic view for views that simply display a template.